docs/interactions/Message_Components.md

```diff
@@ -215,10 +215,22 @@ There are 5 different [select menu components](#DOCS_INTERACTIONS_MESSAGE_COMPON
 
 The string select menu (type `3`) is the *only* select type that allows (and *requires*) apps to define the `options` that appear in the dropdown list. The other 4 select menu components (users, roles, mentionables, and channels) are auto-populated with options corresponding to the resource type—similar to [command option types](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-option-type).
 
-In addition to the `values` array in all [select menu interaction payloads](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-menu-interaction), auto-populated select menu components (users, roles, mentionables, and channels) also include an additional [`resolved` object](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-menu-resolved-object) that provides additional details about the user's selected resource.
+The interaction response for auto-populated select menu components also includes a [`resolved` object](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-menu-resolved-object) that provides additional details about the user's selected resource.
```

Comment:
I think this edit takes clarity away since auto-populated select menu components isn't really defined anywhere else. There's also a lingering "also" that doesn't make sense if you don't have the note about `values`.